src/strategyHelpers/deploymentHelper.ts

```diff
@@ -218,7 +218,12 @@ async function annotateResources(
    )
    if (annotateNamespace) {
       annotateResults.push(
-         await kubectl.annotate('namespace', namespace, annotationKeyValStr)
+         await kubectl.annotate(
+            'namespace',
```

Comment:
again, not sure how reasonable this is but would love to make this value constant